deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -816,6 +816,16 @@ spec:
                       params:
                         type: object
                         description: Map of key-value params for the provider.
+            telemetry:
```

Comment:
Should this be part of the monitoring section?

Reply:
done